holmscan/controller.py

```diff
@@ -22,24 +23,18 @@
 logging.getLogger("anyconfig").setLevel(logging.ERROR)
 
 
-CONFIGURABLES = [
-    "HOLMSCAN_DEBUG",
-    "HOLMSEC_PASSWORD",
-    "HOLMSEC_URL",
-    "HOLMSEC_USERNAME",
-]
+CONFIGURABLES = ["HOLMSCAN_DEBUG", "HOLMSEC_ENDPOINT", "HOLMSEC_TOKEN"]
 DEFAULTS = {
     "HOLMSCAN_DEBUG": False,
-    "HOLMSEC_PASSWORD": "",
-    "HOLMSEC_URL": "https://sc.holmsecurity.com/",
-    "HOLMSEC_USERNAME": "",
+    "HOLMSEC_ENDPOINT": "https://se-api.holmsecurity.com/v1",
+    "HOLMSEC_TOKEN": "",
 }
-REQUIRED = ["HOLMSEC_PASSWORD", "HOLMSEC_URL", "HOLMSEC_USERNAME"]
-VALIDATORS = {"HOLMSEC_URL": "^http(s)?://[a-zA-Z0-9._-]+/$"}
-VALID_EXAMPLES = {"HOLMSEC_URL": "example: https://sc.holmsecurity.com/"}
+REQUIRED = ["HOLMSEC_ENDPOINT", "HOLMSEC_TOKEN"]
```

Comment:
`REQUIRED` is to non descriptive. Required for what?

Reply:
They are required configurations, or else the tool will ask for them. Leaving as is for now.